### DISCUSSION TITLE
Hide muted issues in iOS spotlight results

### DISCUSSION THREAD

**Discussion by ryh** [community/community#8101] [Mobile]
I joined one issue but muted right now, but on iOS I want search app "x" for example.I still got repo with same user(org) named "x", and the issue(own by this user "x") I muted. It's kind of annoying.
So maybe user can configure the setting for what will show in spotlight search results? 

(Github app 1.40.0 on iOS 15.1)
edit: the results is gone on version 1.49.0

**Comment by Vannsansan** [comment]
Hi
